xcpmd/src/acpi-events.c

```diff
@@ -41,12 +41,9 @@ static struct event acpi_event;
 static struct ev_wrapper ** acpi_event_table;
 
 void adjust_brightness(int increase, int force) {
-
-    if ( force || (pm_quirks & PM_QUIRK_SW_ASSIST_BCL) || (pm_quirks & PM_QUIRK_HP_HOTKEY_INPUT)) {
-        if (increase)
-            com_citrix_xenclient_surfman_increase_brightness_(xcdbus_conn, SURFMAN_SERVICE, SURFMAN_PATH);
-        else
-            com_citrix_xenclient_surfman_decrease_brightness_(xcdbus_conn, SURFMAN_SERVICE, SURFMAN_PATH);
+    (void) increase;
+    if (force || (pm_quirks & PM_QUIRK_SW_ASSIST_BCL) || (pm_quirks & PM_QUIRK_HP_HOTKEY_INPUT)) {
+        xcpmd_log(LOG_WARNING, "Brightness adjustment not implemented yet.");
```

Comment:
vglass idl defines increase/decrease_brightness, but it's unclear to me if they are hooked up.